### DISCUSSION TITLE
Firewall Allow ICMP

### DISCUSSION THREAD

**Discussion by corycandia** [Security-Onion-Solutions/securityonion#11460] [2.4] [answered]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

8

### Storage for /

100

### Storage for /nsm

200

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Could someone provide correct guidance/command to allow ICMP response from the local subnet?  I don't know how to add ICMP to a 'customhostgroup' or how to put those on a role.

**Goal**:  I would like each of the nodes to respond to ICMP from a network monitoring instance.

**Example**:  Allow ICMP from 10.0.19.21 or even 10.0.19.0/26.  Network monitoring application gets responses from all SO nodes.  

I did not see any groups in the firewall examples or config file that mentions ICMP.

Documentation talk about adding a host group and host, but what about a protocol or port?
```sudo so-firewall addhostgroup <GROUP_NAME> <IP>```

How do you create a group that allows ICMP?

Can someone Would this be something like:
 ```sudo so-firewall addhostgroup NETWORK_MONITORS```?
```sudo so-firewall includehost NETWORK_MONITORS 10.0.19.21```?
```sudo so-firewall addport NETWORK_MONITORS ICMP```?

Then how do you add this group to each of the roles?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by corycandia** [comment]
Curious if anyone has created custom groups and would post config I could use as an example?

**Comment by TOoSmOotH** [comment, marked as answer]
This is actually a bug. You can manually fix by making the the mod directly to /opt/so/saltstack/default/salt/firewall/iptables.jinja

Will officially be fixed in 2.4.30

https://github.com/Security-Onion-Solutions/securityonion/pull/11496/files
